packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -326,6 +326,151 @@ class CupertinoTextField extends StatefulWidget {
          )),
        super(key: key);
 
+  /// Creates a borderless iOS-style text field.
+  ///
+  /// To provide a prefilled text entry, pass in a [TextEditingController] with
+  /// an initial value to the [controller] parameter.
+  ///
+  /// To provide a hint placeholder text that appears when the text entry is
```

Comment:
It's a bit of a hidden feature. But if there's verbatim doc in various places, you can search for usages of `{@macro ...` in various places

Reply:
Keeping this comment to see if my use of macro was correct. Will close once the documentation tool passes here.